modules/prepare-ocp-for-bare-metal-builds.adoc

```diff
@@ -0,0 +1,199 @@
++
+Check with your cluster administrator to determine exactly which node label should be used. 
+
+. Obtain the Kube API Server's certificate authority (CA) to add to {productname}'s extra certificates. 
+
+.. Enter the following command to obtain the name of the secret containing the CA:
++
+[source,terminal]
+----
+$ oc get sa openshift-apiserver-sa --namespace=openshift-apiserver -o json | jq '.secrets[] | select(.name | contains("openshift-apiserver-sa-token"))'.name
+----
+
+.. Obtain the `ca.crt` key value from the secret in the {ocp} Web Console. The value begins with *"-----BEGIN CERTIFICATE-----"`*. 
+
+.. Import the CA to {productname}. Ensure that the name of this file matches the `K8S_API_TLS_CA` field used in Step 9. 
+
+. Create the following `SecurityContextConstraints` resource for the `ServiceAccount`:
++
+[source,yaml]
+----
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: quay-builder
+priority: null
+readOnlyRootFilesystem: false
+requiredDropCapabilities: null
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+seccompProfiles:
+- '*'
+supplementalGroups:
+  type: RunAsAny
+volumes:
+- '*'
+allowHostDirVolumePlugin: true
+allowHostIPC: true
+allowHostNetwork: true
+allowHostPID: true
+allowHostPorts: true
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: true
+allowedCapabilities:
+- '*'
+allowedUnsafeSysctls:
+- '*'
+defaultAddCapabilities: null
+fsGroup:
+  type: RunAsAny
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: quay-builder-scc
+  namespace: builder
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - quay-builder
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: quay-builder-scc
+  namespace: builder
+subjects:
+- kind: ServiceAccount
+  name: quay-builder
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: quay-builder-scc
+----
+
+. Update the `config.yaml` file of your {productname-ocp} deployment to include an appropriate _bare metal builds_ configuration by using the {ocp} web console.
+
+.. Click *Operators* -> *Installed Operators* -> *Red Hat Quay* -> *Quay Registry*. 
+
+.. Click the name of your registry, for example, *example-registry*. 
+
+.. Under *Config Bundle Secret*, click the name of your configuration bundle, for example, *extra-ca-certificate-config-bundle-secret*. 
+
+.. Click *Actions* -> *Edit Secret*.
+
+.. Add the following information to your {productname} `config.yaml` file, replacing each value with information that is relevant to your specific installation:
++
+[source,yaml]
+----
+FEATURE_USER_INITIALIZE: true
+BROWSER_API_CALLS_XHR_ONLY: false
+SUPER_USERS:
+- <superusername>
+FEATURE_USER_CREATION: false
+FEATURE_QUOTA_MANAGEMENT: true
+FEATURE_BUILD_SUPPORT: True
+BUILD_MANAGER:
```

Comment:
`BUILDMAN_HOSTNAME` is missing
```
BUILDMAN_HOSTNAME: ${BUILDMAN_HOSTNAME}:443
BUILD_MANAGER:
- ephemeral
- ALLOWED_WORKER_COUNT: 10
```
How to get the value of `${BUILDMAN_HOSTNAME}`  is
```
$ oc get route quayregistry-quay-builder -n ${QUAY_PROJECT} -o jsonpath='{.spec.host}'
```

Reply:
Added.